mtp_cashbook/apps/cashbook/tasks.py

```diff
@@ -111,10 +111,10 @@ def check_balance_is_below_cap(prison, prisoner_number):
             for account in NOMIS_ACCOUNTS
         ), 'not all response values are natural ints'
     except AssertionError as e:
-        logger.exception(f'NOMIS balances for {prisoner_number} is malformed: {e}')
+        logger.exception('NOMIS balances is malformed', {'prisoner_number': prisoner_number, 'exception': e})
     except requests.RequestException:
-        logger.exception(f'Cannot lookup NOMIS balances for {prisoner_number}')
+        logger.exception('Cannot lookup NOMIS balances for prisoner', {'prisoner_number': prisoner_number})
```

Comment:
I think @ushkarev mentioned that these exceptions/emails are counted to keep track of the number of how often this occurs?

I guess is still possible to count them, they'd just be grouped in GMail so I don't imagine this being a problem but I thought I'd mention just-in-case ☺️